src/components/molecules/MoleculeSRPCard.vue

```diff
@@ -76,121 +91,138 @@ export default {
-}
-.card-title {
-    font-size: 1rem;
-    grid-column: 1 / 5;
-    grid-row: 1 / 1;
-    -webkit-box-orient: vertical;
-    display: -webkit-box;
-    -webkit-line-clamp: 1;
-    overflow: hidden;
-}
-
-.card-location {
-    align-items: center;
-    display: flex;
-    font-size: var(--sp-size-sm);
-    gap: 10px;
-    grid-column: 1 / 5;
-    grid-row: 2 / 2;
     position: relative;
-    line-height: 1.1;
-    color: #6b6b6b;
-}
-
-.location-icon {
-    font-size: 20px;
-}
-
-.location-text {
-    -webkit-box-orient: vertical;
-    display: -webkit-box;
-    -webkit-line-clamp: 2;
-    overflow: hidden;
-}
-
-.card-rating {
-    grid-column: 1 / 3;
-    grid-row: 3 / 3;
-    margin-top: 0.75rem;
-}
-
-.card-distance {
-    background-color: rgb(214, 253, 255);
-    width: 58px;
-    text-align: center;
-    padding: 0.75rem 0.75rem;
-    border-radius: var(--border-default);
-    grid-column: 5 / 7;
-    grid-row: 1 / 3;
-    align-self: end;
-    justify-self: end;
-}
-
-.distance-text {
-    line-height: var(--lh-small);
-    font-weight: var(--bold-font);
-    font-size: 16px;
-}
-
-.text-color {
-    color: var(--secondary-color);
-}
-
-.card-rate {
-    align-items: center;
-    display: flex;
-    font-weight: var(--bold-font);
-    gap: 10px;
-    grid-column: 1 / 4;
-    grid-row: 4 / 5;
-    margin-bottom: 0.75rem;
-}
-
-.rate-icon {
-    color: green;
-}
-
-.card-spot {
-    grid-column: 4 / 6;
-    grid-row: 4 / 5;
-}
-
-.card-spot-red {
-    color: red;
-    font-weight: var(--bold-font);
-}
-
-.card-spot-green {
-    color: green;
-    font-weight: var(--bold-font);
-}
-
-.card-btn {
-    grid-column: 1 / 7;
-    grid-row: 5 / 6;
-    font-weight: var(--bold-font);
+    transition: transform 0.3s ease;
+    cursor: pointer;
+
+    &:hover {
+        transform: scale(1.03);
+
+        &::before {
+            content: '';
+            position: absolute;
+            background: var(--secondary-color);
+            bottom: 25%;
+            left: 0;
+            width: 2px;
+            height: 50%;
+        }
+
+        &::after {
+            content: '';
+            position: absolute;
+            background: var(--secondary-color);
+            right: 0;
+            bottom: 25%;
+            width: 2px;
+            height: 50%;
+        }
+    }
+
+    .card-primary {
+        display: flex;
+        justify-content: space-between;
+        gap: 20px;
+        align-items: flex-start;
+    }
+
+    .card-secondary {
+        display: flex;
+        justify-content: space-between;
+        margin-top: 14px;
+    }
+
+    .card-title {
+        font-size: 15px;
+        font-weight: 500;
+        -webkit-box-orient: vertical;
+        display: -webkit-box;
+        -webkit-line-clamp: 1;
+        overflow: hidden;
+    }
+
+    .card-location {
+        align-items: center;
+        display: flex;
+        font-size: 12px;
+        gap: 5px;
+        position: relative;
+        line-height: 1.1;
+        color: #6b6b6b;
```

Comment:
why a new color code.
Isn't this defined as a a variable like other colors

Reply:
this is some other hexcode ...not defined in variable.